src/app/sign-in/page.tsx

```diff
@@ -69,7 +69,7 @@ const SignInPage = (): JSX.Element => {
     supabase.auth.signInWithOAuth({
       provider: 'google',
       options: {
-        redirectTo: 'http://localhost:3000/auth/callback'
+        redirectTo: '/auth/callback'
```

Comment:
just having the path, rather than the hostname _and_ the path, makes it so that this will work whether it's deployed to `getspooky.net` or just running on `localhost`